app/entry.py

```
@@ -0,0 +1,357 @@
+                logger.error("❌ Emergency backup also failed: %s", backup_error)
+
+    async def _backup_loop(self):
+        """Background task that periodically creates emergency backups."""
+        try:
+            while self.is_running:
+                await asyncio.sleep(self.config.backup_interval_seconds)
+                current_time = datetime.now(timezone.utc)
+                if (
+                    current_time - self.last_backup_time
+                ).total_seconds() >= self.config.backup_interval_seconds:
+                    if len(self.emergency_backup) > 0:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)